src/perform_step/low_order.jl

```diff
@@ -249,6 +249,8 @@ end
   du₁ = integrator.f(uprev,p,t)
   L = integrator.g(uprev,p,t)
   mil_correction = zero(u)
+  ggprime_norm = zero(eltype(u)) #0
+  # sqdt = integrator.sqdt
```

Comment:
what's the reason here?

Reply:
no, the `sqdt=integrator.sqdt` getting commented, and then using the integrator in the other lines? Seems like an odd change.